.gitignore

```diff
@@ -97,3 +97,5 @@ tmp.*.rs
 version.md
 version.ml
 version.texi
+src/libcore/Cargo.lock
```

Comment:
I'm not sure this change is okay. I don't know what rust core people think about this.

cc @rust-lang/core